README.md

```diff
@@ -164,12 +165,18 @@ __onnx_coreml_input_shape_map__: dict (str: List[int])
     0: Sequence, 1: Batch, 2: channel, 3: height, 4: width.  
     For example, an input of rank 2 could be mapped as [3,4] (i.e. H,W) or [1,2] (i.e. B,C) etc.  
 
-__disable_coreml_rank5_mapping__: bool  
-	  If True, then it disables the "RANK5_ARRAY_MAPPING" or enables the "EXACT_ARRAY_MAPPING"
-        option in CoreML (https://github.com/apple/coremltools/blob/655b3be5cc0d42c3c4fa49f0f0e4a93a26b3e492/mlmodel/format/NeuralNetwork.proto#L67)
-        Thus, no longer, onnx tensors are forced to map to rank 5 CoreML tensors.
-        With this flag on, a rank r ONNX tensor, (1<=r<=5), will map to a rank r tensor in CoreML as well.
-        This flag must be on to utilize any of the new layers added in CoreML 3 (i.e. specification version 4, iOS13)
+__target_ios__: str
+      Target Deployment iOS Version (default: '12')
+      Supported iOS version options: '11.2', '12', '13'
+      Model conversion will use target iOS version to convert model accoding to iOS and CoreML version support.
+      iOS 11.2 (CoreML 0.8) does not support resize_bilinear, crop_resize layers 
+      - (Supported features: https://github.com/apple/coremltools/releases/tag/v0.8)
+      iOS 12 (CoreML 2.0)
+      - (Supported features: https://github.com/apple/coremltools/releases/tag/v2.0)
+      iSO 13 (CoreML 3.0)
+      - (Supported features: https://github.com/apple/coremltools/releases/tag/3.0-beta6)
+      
+      For iOS 11 and 12, RANK5 mapping is required and will be used. iOS 13 onwards, ND Tensor support is added.
```

Comment:
This sentence about rank mapping can be removed for now. Later if we feel the need, can be added back.